src/Sil/Bundle/ContactBundle/Resources/views/block/address-block.html.twig

```diff
@@ -0,0 +1,58 @@
+<li class="item contact-address"
+    data-id="{{ address.id }}"
+    data-edit-url="{{ path('sil_contact_address_edit', {'id': address.id}) }}"
+    data-remove-url="{{ path('sil_contact_address_delete', {'id': address.id, 'list': true}) }}"
+    data-make-default-url="{{ path('sil_contact_set_default_address', {'addressId': address.id}) }}"
+>
+    <div class="ui center aligned stackable grid">
+        <div class="six wide column">
+            <div class="content">
+                <div class="header">
+                    {{ address.type }}&nbsp;
+                </div>
+                <div class="description">
+                    <i class="blue address card icon"></i>
+                    {{ address.street }}
+                    <div>
+                        {{ address.city.postCode }}
+                        {{ address.city.name }}
+                        {{ address.country }}
+                    </div>
+                </div>
+            </div>
+        </div>
+        {% if isDefaultAddress %}
+            <div id="default-address" class="four wide column">
+                <span class="ui small teal label">{{ 'Main'|trans }}</span>
+            </div>
+        {% else %}
+            <div id="default-address" class="four wide column">
+                <a
+                    class="ui mini label make-default"
+                    data-tooltip="{{'Set as main address'|trans}}"
```

Comment:
You should use structured translations as we have decided to apply on our bundles (see example https://github.com/sil-project/Platform/blob/7ad411806a0d05cff67e0962c524d6616974acf4/src/Sil/Bundle/ProductBundle/Resources/translations/messages.fr.yml)